File: ImagemathStudent/ImagemathStudent/QnA/Question.swift
//
//  Question.swift
//  ImagemathStudent
//
//  Created by 정해민 on 2020/09/09.
//  Copyright © 2020 정해민. All rights reserved.
//

import Foundation


struct Question{
    let questionSeq:Int
    let title:String
    let contents:String
    let postTime:Int
    let updateTime:Int
    let uploaderSeq:Int
    
}
extension Question{
    
    static func getDummyData()->[Question]{
        return [
            Question(questionSeq: 0, title: "이미지 포함 테스트", contents: "테스트", postTime: 1515, updateTime: 1515, uploaderSeq: 0),
            Question(questionSeq: 1, title: "이미지 포함 테스트", contents: "테스트", postTime: 1515, updateTime: 1515, uploaderSeq: 0),
            Question(questionSeq: 2, title: "이미지 포함 테스트", contents: "테스트", postTime: 1515, updateTime: 1515, uploaderSeq: 0),
            Question(questionSeq: 3, title: "이미지 포함 테스트", contents: "테스트", postTime: 1515, updateTime: 1515, uploaderSeq: 0),
            Question(questionSeq: 4, title: "이미지 포함 테스트", contents: "테스트", postTime: 1515, updateTime: 1515, uploaderSeq: 0)
        ]
    }
}

File: android/ImageMathStudent/app/src/main/java/com/haemin/imagemathstudent/Data/Question.java
package com.haemin.imagemathstudent.Data;

import com.google.gson.annotations.SerializedName;

public class Question {

    @SerializedName("questionSeq")
    String questionSeq;
    @SerializedName("title")
    String title;
    @SerializedName("contents")
    String contents;
    @SerializedName("postTime")
    long postTime;
    @SerializedName("updateTime")
    long updateTime;
    @SerializedName("uploaderSeq")
    String uploaderSeq;

    public String getQuestionSeq() {
        return questionSeq;
    }

    public void setQuestionSeq(String questionSeq) {
        this.questionSeq = questionSeq;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getContents() {
        return contents;
    }

    public void setContents(String contents) {
        this.contents = contents;
    }

    public long getPostTime() {
        return postTime;
    }

    public void setPostTime(long postTime) {
        this.postTime = postTime;
    }

    public long getUpdateTime() {
        return updateTime;
    }

    public void setUpdateTime(long updateTime) {
        this.updateTime = updateTime;
    }

    public String getUploaderSeq() {
        return uploaderSeq;
    }

    public void setUploaderSeq(String uploaderSeq) {
        this.uploaderSeq = uploaderSeq;
    }
}
